tests/bluetooth/tester/src/gatt.c

```diff
@@ -2061,6 +2063,22 @@ static void get_attr_val(uint8_t *data, uint16_t len)
 	}
 }
 
+static void connect(uint8_t *data, uint16_t len)
```

Comment:
I think `eatt_connect()` would be a clearer name, not just because it doesn't conflict with the standard POSIX `connect()` API name, but also because it'd more accurately describe the purpose of the function.